app/views/layouts/_header.html.erb

```diff
@@ -18,7 +18,7 @@
     </div>
     <div class="govuk-header__content">
 
-      <a href="#" class="govuk-header__link govuk-header__link--service-name">
+      <a href="<%= providers_legal_aid_applications_url %>" class="govuk-header__link govuk-header__link--service-name">
```

Comment:
**[not a blocker/nit pick]** 💭  If we're regulary embedding ERB code into the raw HTML, specially in these link cases, might be worth using the Rails helpers instead (e.g. `link_to`).

⚠️ Also, I'm not sure if this is obvious for everyone, but this URL should **be only for providers**, so if you're on the **citizen/applicant side** of the application the URL **should be different**, as far as I'm aware.

Reply:
💭 Might be worth considering adjusting the layout to fork between **providers** and **applicant**.